rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -89,6 +94,13 @@ GEMFILE
 %doc %{gem_instdir}/LICENSE
 %doc %{gem_instdir}/README.md
 
+%posttrans
+%{foreman_db_migrate}
```

Comment:
not related to apipie but other packages run the migration and seed here too so I added it to be consistent